src/Spout/Reader/XLSX/RowIterator.php

```diff
@@ -220,9 +220,8 @@ protected function processDimensionStartingNode($xmlReader)
     {
         // Read dimensions of the sheet
         $dimensionRef = $xmlReader->getAttribute(self::XML_ATTRIBUTE_REF); // returns 'A1:M13' for instance (or 'A1' for empty sheet)
-        if (preg_match('/[A-Z\d]+:([A-Z\d]+)/', $dimensionRef, $matches)) {
-            $lastCellIndex = $matches[1];
-            $this->numColumns = CellHelper::getColumnIndexFromCellIndex($lastCellIndex) + 1;
+        if (preg_match('/^[A-Z]+\d+:([A-Z]+\d+)$/', $dimensionRef, $matches)) {
```

Comment:
I would remove the `^` and `$` as the dimension ref may contain leading or trailing spaces. Also, since this regex is run only once per sheet, the perf gain is not noticeable.
The change to the regex is fine though.